### PR TITLE
[bitnami/*]Modified the pipelines to contain breaking changes

### PR DESCRIPTION
**Pull request by superaleks**
Signed-off-by: Aleksandra Lukic <alukic@vmware.com>

**Description of the change**

Change the pipelines to match the breaking change.

**Checklist**

- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by beltran-rubo — APPROVED** [review]

**Comment by beltran-rubo** [comment]
LGTM. Please @javsalgar check this PR as well before merging it.

File: bitnami/wordpress/README.md [hunk 1]
```diff
@@ -626,4 +626,4 @@ Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
-limitations under the License.
```
Comment (javsalgar) [inline bitnami/wordpress/README.md:629]: Now that we saw that the action works, could you undo this change?

**Review by javsalgar — COMMENTED** [review]
LGTM, could you check my comment?

File: bitnami/wordpress/README.md (hunk 1 above)
Comment (superaleks) [reply bitnami/wordpress/README.md]: Yes, done!
